src/main/kotlin/io/learning/api/di/AppConfigModule.kt

```diff
@@ -18,6 +18,7 @@ class AppConfigModule: AbstractModule() {
     @Provides
     @Singleton
     fun getVertx(): Vertx {
+        System.setProperty("vertx.logger-delegate-factory-class-name", "io.vertx.core.logging.Log4j2LogDelegateFactory")
```

Comment:
maybe you can set this somewhere else? I wouldn't expect to find it here

Reply:
True, I just need it to be set before the vertx is created. Moved it now.